extensions/sql-migration/src/wizard/skuRecommendation/assessmentSummaryCard.ts

```diff
@@ -387,17 +387,21 @@ export class AssessmentSummaryCard implements vscode.Disposable {
 
 	// Used to update the summary of assessment card later after its initialization.
 	// TODO - We can remove dummayData input later and use a some other way to pass the values. Once we start implementing the whole page.
-	public async updateContent(dummyData: DummyData) {
-		await this._assessmentResultText.updateProperties({ "value": constants.ASSESSED_DBS(dummyData.assessmentResult) });
-		await this._readyText.updateProperties({ "value": dummyData.ready.toString() });
-		await this._needsReviewText.updateProperties({ "value": dummyData.needsReview.toString() });
-		await this._notReadyText.updateProperties({ "value": dummyData.notReady.toString() });
-		await this._blockersText.updateProperties({ "value": dummyData.blockers.toString() });
-		await this._warningsText.updateProperties({ "value": dummyData.warnings.toString() });
-
-		await this._recommendedConfigurationText.updateProperties({ "value": dummyData.skuRecommendation.toString() });
-		await this._vmRecommendedConfigurationText.updateProperties({ "value": dummyData.vmRecommendation });
+	public async updateAssessmentResult(
+		databaseCounts: number, ready: number, needsReview: number, notReady: number,
+		blockers: number, warnings: number) {
+		await this._assessmentResultText.updateProperties({ "value": constants.ASSESSED_DBS(databaseCounts) });
+		await this._readyText.updateProperties({ "value": ready.toString() });
+		await this._needsReviewText.updateProperties({ "value": needsReview.toString() });
```

Comment:
recommend using text-control.value("newstring") property to set values instead of updateProperteis method

Reply:
U mean using text-control.value = "newstring" directly;

Reply:
yes, I recommend direct assignment to the variable.
sorry, was thinking of observable type variable in the example.

Reply:
Done. Used direct assignment to variables.